ui/src/app/views/workspace-edit/component.css

```diff
@@ -16,3 +16,11 @@
 .checkbox-group {
   padding: 1rem 0;
 }
+
+.red-background {
+  background-color: #FFEEEE;
+}
+
+.field {
```

Comment:
Would selecting `form > div` (or replacing this with `form p`) work here? Just looking to avoid extra markup in the template if we can derive what we want from the structure or from semantic tags.

Reply:
I can do `div.form-area > div` (since that's what we're using for formatting with project clarity). Using paragraph tags didn't work and threw a template parse error. Either way I think `div.form-area > div` is probably fine.